src/domd_cfg.c

```diff
@@ -377,21 +377,21 @@ static uint32_t domd_irqs[] = {
 struct xen_domain_cfg domd_cfg = {
 	.mem_kb = 2097152,
 
-	.flags = (XEN_DOMCTL_CDF_hvm | XEN_DOMCTL_CDF_hap | XEN_DOMCTL_CDF_iommu),
+	.flags = (XEN_DOMCTL_CDF_hvm | XEN_DOMCTL_CDF_hap),
 	.max_evtchns = 10,
 	.max_vcpus = 4,
 	.gnt_frames = 32,
 	.max_maptrack_frames = 1,
 
 	.iomems = domd_iomems,
-	.nr_iomems = sizeof(domd_iomems) / sizeof(*domd_iomems),
+	.nr_iomems = 0 /* sizeof(domd_iomems) / sizeof(*domd_iomems) */,
```

Comment:
Please remove comments. And why are you changing domd config? Accoding to the commit message - you are fixing domu

Reply:
This commit has been done only for demonstration purposes - to boot any domain under Qemu. These changes are not about to left in main branch. Final goal is to boot DomD on some HW, where all of these structs will be filled with proper values.

Reply:
I see. Then I'd rather change name to WIP and leave a comment that those changes should be used for the future implementations.

Reply:
@oleksiimoisieiev this PR will be closed and commit moved to xentroops WIP branches, thanks for review

Reply:
ok